pkg/acs/calacs/lib/acshist.c

```diff
@@ -160,39 +162,63 @@ int blevHistory (ACSInfo *acs, Hdr *phdr, int done, int driftcorr) {
 	addHistoryKw (phdr, "  Overscan region table: ");
 	if (TabHistory (&acs->oscn, phdr))
 		return (status);
-  
+
 	if (hstio_err())
     return (status = HEADER_PROBLEM);
-  
+
 	return (status);
 }
 
+int sinkHistory (ACSInfo *acs, Hdr *phdr) {
```

Comment:
``*acs`` should be ``const``. I think ``*phdr`` should be also but this would require a bit of a refactor so leave ``*phdr`` for now.